include/verilated_fst_c.cpp

```diff
@@ -213,26 +218,60 @@ void VerilatedFst::declDouble(vluint32_t code, const char* name, int dtypenum, f
 // so always inline them.
 
 VL_ATTR_ALWINLINE
-void VerilatedFst::emitBit(vluint32_t code, vluint32_t newval) {
+void VerilatedFst::emitB(vluint32_t code, CData newval) {
```

Comment:
I think the longer name here & elsewhere is easier to understand, but I understand why you want to rename some to indicate the data types.

```suggestion
void VerilatedFst::emitBit(vluint32_t code, CData newval) {
```